.env.stage

```diff
@@ -22,6 +22,16 @@ REACT_APP_ETHEREUM_ERC20_TOKEN_ADDRESS="0x0b10e304088b2BA2B2acfD2f72573FAaD31a13
 REACT_APP_ETHEREUM_ERC20_NAME="CERENETWORK"
 REACT_APP_ETHEREUM_ERC20_SYMBOL="CERE"
 
+REACT_APP_POLYGON_CHAIN_ID="81"
+REACT_APP_POLYGON_NETWORK_ID="80001"
+REACT_APP_POLYGON_NETWORK_NAME="Polygon Mumbai"
+REACT_APP_POLYGON_BRIDGE_ADDRESS="0xBDc040b15561CFC9ED1Ce049DE8bEAbf0b520b33"
+REACT_APP_POLYGON_ERC20_HANDLER_ADDRESS="0x8Ee8876d13e79b846fb6c3e5Ffe226a2e111387a"
+REACT_APP_POLYGON_RPC_URL=""
+REACT_APP_POLYGON_ERC20_TOKEN_ADDRESS="0xd111d479e23A8342A81ad595Ea1CAF229B3528c3"
+REACT_APP_POLYGON_ERC20_NAME="CN"

```

Comment:
Use the same values as in the Ethereum network here.

Reply:
updated